doc/source/advanced/executor.rst

```diff
@@ -2,16 +2,26 @@
 Execution Engine
 ****************
 
-Fetching the data required for generating visualizations can be computationally expensive, especially on large datasets. Lux provides a extensible framework for users to pick their own execution backend for data processing. We currently support Pandas (default, :mod:`lux.executor.PandasExecutor`) and SQL (:mod:`lux.executor.SQLExecutor`). In this tutorial, we explain how to use switch to SQL as an execution backend, as an example of how you can use a different data processing mechanism in Lux.
+Lux provides an extensible framework for users to pick their own execution backend for data processing. We currently support Pandas (:mod:`lux.executor.PandasExecutor`) and SQL (:mod:`lux.executor.SQLExecutor`) as the execution engine. By default, Lux leverages Pandas as its execution backend; in other words, the data processing code is performed as a set of Pandas operations on top of dataframe. In this tutorial, we further explain how Lux can be used with SQL with tables inside a Postgres database.
 
 Please refer to :mod:`lux.executor.Executor`, if you are interested in extending Lux for your own execution backend.
 
-
-
 SQL Executor
 =============
 
-Lux extends its visualization exploration operations to data within SQL databases. By using the SQL Executor, users can specify a SQL database to connect a LuxSQLTable for generating all the visualizations recommended in Lux.
+Lux extends its visualization capabilities to SQL within Postgres databases. By using the SQLExecutor, users can create a :code:`LuxSQLTable` that connects to a Postgres database. When the :code:`LuxSQLTable` object is printed out, Lux displays a subset of the data and recommends a default set of visualizations to display.
+
+.. image:: https://github.com/lux-org/lux/blob/sql-engine/doc/source/img/SQLexecutor.gif?raw=true
```

Comment:
Also the GIF seems to be based on an old version of the frontend, since the preview caption and buttons are different. Can we do a new recording of the GIF with several changes:
1) Update the frontend to be based on latest version of code
2) We don't need to show the data upload cell, it would be good to just zoom in to the statement on where the LuxSQLTable is created. Something similar to this, where the import, pycopg2 connection, and LuxSQLTable declaration is separated into three cells:
```python
import lux
import pandas as pd
```

```python
import psycopg2
connection = psycopg2.connect("host=localhost dbname=testdb user=testuser password=testpass")
lux.config.set_SQL_connection(connection)
```


```python
tbl = lux.LuxSQLTable("college")
tbl
```
![image](https://user-images.githubusercontent.com/5554675/114978523-e0a64c80-9ebb-11eb-80ef-500bc82a74a4.png)
3) The GIF doesn't need to spend a lot of time scrolling through and clicking on the tabs of the widget, simply Toggle to the Lux view and clicking on one tab would be sufficient.